source/jormungandr/jormungandr/default_settings.py

```diff
@@ -1,35 +1,40 @@
 # encoding: utf-8
 
 import logging
+import os
+import json
 
 # path of the configuration file for each instances
-INSTANCES_DIR = '/etc/jormungandr.d'
+INSTANCES_DIR = os.getenv('JORMUNGANDR_INSTANCES_DIR', '/etc/jormungandr.d')
 
 # Start the thread at startup, True in production, False for test environments
 START_MONITORING_THREAD = True
 
 #URI for postgresql
 # postgresql://<user>:<password>@<host>:<port>/<dbname>
 #http://docs.sqlalchemy.org/en/rel_0_9/dialects/postgresql.html#psycopg2
-SQLALCHEMY_DATABASE_URI = 'postgresql://navitia:navitia@localhost/jormungandr'
+SQLALCHEMY_DATABASE_URI = os.getenv('JORMUNGANDR_SQLALCHEMY_DATABASE_URI', 'postgresql://navitia:navitia@localhost/jormungandr')
 
-DISABLE_DATABASE = False
+DISABLE_DATABASE = os.getenv('JORMUNGANDR_DISABLE_DATABASE', False)
 
 # disable authentication
-PUBLIC = True
+PUBLIC = os.getenv('JORMUNGANDR_IS_PUBLIC', True)
 
 #message returned on authentication request
-HTTP_BASIC_AUTH_REALM = 'Token Required'
+HTTP_BASIC_AUTH_REALM = os.getenv('JORMUNGANDR_HTTP_BASIC_AUTH_REALM', 'Token Required')
 
 from jormungandr.logging_utils import IdFilter
 
+log_level = os.getenv('JORMUNGANDR_LOG_LEVEL', 'DEBUG')
+log_format = os.getenv('JORMUNGANDR_LOG_FORMAT', '[%(asctime)s] [%(request_id)s] [%(levelname)5s] [%(process)5s] [%(name)10s] %(message)s')
+
 # logger configuration
 LOGGER = {
```

Comment:
It will require a json, and I want to avoid having json in a environment variable, it's ugly.
We almost never update the logger configuration